### ISSUE TITLE
OSI Ground Truth export file misses Signal's type

### ISSUE THREAD

**Issue by akardashdxc** [esmini/esmini#430]
Hi.
We are currently working with OSI Ground Truth files generated by `--osi_file` attribute and found a discrepancy.

- The code segment in method [bool OpenDrive::LoadOpenDriveFile(const char* filename, bool replace)](https://github.com/esmini/esmini/blob/master/EnvironmentSimulator/Modules/RoadManager/RoadManager.cpp#L3979) requires Signal's type, subtype and value be devided by delimeter `.`.
- Method [bool OpenDrive::LoadSignalsByCountry(const std::string& country)](https://github.com/esmini/esmini/blob/master/EnvironmentSimulator/Modules/RoadManager/RoadManager.cpp#L5625) reads type, subtype and value from mapping file (in our case [.\resources\traffic_signals\de_traffic_signals.txt](https://github.com/esmini/esmini/blob/master/resources/traffic_signals/de_traffic_signals.txt) "as is". This data is not separated by delimeter `.`.

This leads to a problem when OpenDrive Signal's type can not be coverted to OSI's type using "de" country. Changing `350-10=TYPE_ZEBRA_CROSSING` with `350.-1.0=TYPE_ZEBRA_CROSSING` in de_traffic_signals.txt fixes the issue. It looks like the whole mapping file is incorrect and requires rework.
Also it is not 100% clear if `350-10` means type="350", subtype="-10" and value="" OR subtype="-1" and value="0" OR subtype="" and value="10". If the last variant is correct, then de_traffic_signals.txt is completely correct and we just have to adjust our xodr file to this subtype and value.

By the way, this mapping file misses Signal with type="293" which also corresponds to pedestrian crosswalk. Not the Sign, but the rectangle on the road with markings. Would be great if you also add it to the file.

Scenario example: [Crosswalk.zip](https://github.com/esmini/esmini/files/11423013/Crosswalk.zip)

Thank you :)

**Comment by eknabevcc** [comment]
In the [German system](https://en.wikipedia.org/wiki/Road_signs_in_Germany#Directional_signs) I see that the crosswalk signs have ID either 350-10 or 350-20. 

![cw](https://github.com/esmini/esmini/assets/29146839/5237d569-4633-48d7-83b4-ca2d98c3039b)

In esmini mapping the 350 is the type and 10/20 the value. I see in provided xodr that type is the expected 350, but value is 0, not 10 or 20. Hence no mapping is found.

The esmini road sign mapping system is described here: [User guide - Road signs](https://esmini.github.io/#_road_signs)

That said, thanks to your report I found an issue: To verify that a valid value would work, I modified your xodr, changing value 0 into 10. Still it didn't work. Root cause being that provided subtype ("-1") was parsed and used as is. esmini then tried to load de_350_none-10.osgb - which clearly does not exist. 

Checking [OpenDRIVE standard](https://www.asam.net/index.php?eID=dumpFile&t=f&f=4422&token=e590561f3c39aa2260e5442e29e93f6693d1cccd#top-016f925e-bfe2-481d-b603-da4524d7491f) I found this statement: "Subtype identifier according to country code or "-1" / "none""

Now, by commit 676685cb5b869e3e78a8bafb560a1f13ed0e50bb at least this specific issue has been solved, i.e. ignoring "none" and "-1" as subtype. These values having the same effect as leaving subtype out altogether.

The fix is merged on dev branch. Changing "value" into 10 in the provided xodr will at least make esmini try to load "de_350-10.osgb" which is the expected behavior. 

I haven't verified that it propagates to OSI, but I assume it does. Otherwise let us know and we'll investigate further.
